### PR TITLE
e2e(angular15-microfrontends-lazy-components sample)

### PR DESCRIPTION
**Pull request by PeaceDate**
Screenshot 2022-12-21 at 18 43 32

### CODE REVIEW

**Review by zackarychapple — APPROVED** [review]

**Comment by zackarychapple** [comment]
This does resolve the issue around the page constantly refreshing. The data in the table does not load, I'm not sure if that is considered the app not loading correctly though.  Hopefully @timonkrebs can take a peek.

File: cypress/common/selectors.ts [hunk 1]
```diff
@@ -78,8 +78,8 @@ export const blocks = {
 }
 
 export const fields = {
-    nameField: '[data-e2e="FORM__FILED_NAME"]',
-    emailField: '[data-e2e="FORM__FILED_EMAIL"]',
+    nameField: '[data-e2e="FORM__FIELD_NAME"]',
```
Comment (edchai) [inline cypress/common/selectors.ts]: please remove both and use commonField

File: angular15-microfrontends-lazy-components/e2e/checkAngular15MicroLazyComponentsApps.cy.ts [hunk 2]
```diff
@@ -0,0 +1,542 @@
...
+        tableHeaderText: Constants.elementsText.mdmfAppTableHeader,
+        isCardBody: true,
+        sharedHeader: Constants.elementsText.mdmfSharedHeader,
+        sharedParagraph: Constants.commonPhrases.mdmfSharedParagraph,
+        path: Constants.elementsText.mdmfProfile.path,
+        host: 4200
+    }
+]
+
+function addUser(name: string, email: string) {
```
Comment (edchai) [inline angular15-microfrontends-lazy-components/e2e/checkAngular15MicroLazyComponentsApps.cy.ts]: why function on this level?

File: angular15-microfrontends-lazy-components/e2e/checkAngular15MicroLazyComponentsApps.cy.ts [hunk 3]
```diff
@@ -0,0 +1,542 @@
...
+const appsData = [
+    {
+        appNameText: Constants.elementsText.mdmfShell.name,
+        headerText: Constants.elementsText.mdfmShellHeader,
+        isWelcomeText: true,
+        welcomeText: Constants.commonPhrases.mdmfShellWelcome.replace('11', '15'),
+        paragraphText: Constants.commonPhrases.mdmfShellParagraph,
+        tableHeaderText: Constants.elementsText.mdmfAppTableHeader,
+        isCardBody: false,
+        sharedHeader: '',
```
Comment (edchai) [inline angular15-microfrontends-lazy-components/e2e/checkAngular15MicroLazyComponentsApps.cy.ts]: not quite sure about it, what is the point of empty field check?

**Review by edchai — CHANGES_REQUESTED** [review]

File: angular15-microfrontends-lazy-components/e2e/checkAngular15MicroLazyComponentsApps.cy.ts (hunk 3 above)
Comment (PeaceDate) [reply angular15-microfrontends-lazy-components/e2e/checkAngular15MicroLazyComponentsApps.cy.ts]: It's only for the if statement, cause my apps is different a little

File: cypress/common/selectors.ts (hunk 1 above)
Comment (PeaceDate) [reply cypress/common/selectors.ts]: Fixed

File: angular15-microfrontends-lazy-components/e2e/checkAngular15MicroLazyComponentsApps.cy.ts (hunk 2 above)
Comment (PeaceDate) [reply angular15-microfrontends-lazy-components/e2e/checkAngular15MicroLazyComponentsApps.cy.ts]: Fixed

File: angular15-microfrontends-lazy-components/e2e/checkAngular15MicroLazyComponentsApps.cy.ts (hunk 3 above)
Comment (PeaceDate) [reply angular15-microfrontends-lazy-components/e2e/checkAngular15MicroLazyComponentsApps.cy.ts]: Fixed

**Review by edchai — APPROVED** [review]

**Pull request merged**